pkg/operator/sync_openshiftcontrollermanager_v311_00.go

```diff
@@ -38,43 +37,11 @@ func syncOpenShiftControllerManager_v311_00_to_latest(c OpenShiftControllerManag
 	errors := []error{}
 	var err error
 	operatorConfig := originalOperatorConfig.DeepCopy()
-	clientHolder := resourceapply.NewKubeClientHolder(c.kubeClient)
-	directResourceResults := resourceapply.ApplyDirectly(clientHolder, c.recorder, v311_00_assets.Asset,
-		"v3.11.0/openshift-controller-manager/informer-clusterrole.yaml",
-		"v3.11.0/openshift-controller-manager/informer-clusterrolebinding.yaml",
-		"v3.11.0/openshift-controller-manager/ingress-to-route-controller-clusterrole.yaml",
-		"v3.11.0/openshift-controller-manager/ingress-to-route-controller-clusterrolebinding.yaml",
-		"v3.11.0/openshift-controller-manager/tokenreview-clusterrole.yaml",
-		"v3.11.0/openshift-controller-manager/tokenreview-clusterrolebinding.yaml",
-		"v3.11.0/openshift-controller-manager/leader-role.yaml",
-		"v3.11.0/openshift-controller-manager/leader-rolebinding.yaml",
-		"v3.11.0/openshift-controller-manager/ns.yaml",
-		"v3.11.0/openshift-controller-manager/old-leader-role.yaml",
-		"v3.11.0/openshift-controller-manager/old-leader-rolebinding.yaml",
-		"v3.11.0/openshift-controller-manager/separate-sa-role.yaml",
-		"v3.11.0/openshift-controller-manager/separate-sa-rolebinding.yaml",
-		"v3.11.0/openshift-controller-manager/sa.yaml",
-		"v3.11.0/openshift-controller-manager/svc.yaml",
-		"v3.11.0/openshift-controller-manager/servicemonitor-role.yaml",
-		"v3.11.0/openshift-controller-manager/servicemonitor-rolebinding.yaml",
-		"v3.11.0/openshift-controller-manager/buildconfigstatus-clusterrole.yaml",
-		"v3.11.0/openshift-controller-manager/buildconfigstatus-clusterrolebinding.yaml",
-	)
-	resourcesThatForceRedeployment := sets.NewString("v3.11.0/openshift-controller-manager/sa.yaml")
-	forceRollout := false
-
-	for _, currResult := range directResourceResults {
-		if currResult.Error != nil {
-			errors = append(errors, fmt.Errorf("%q (%T): %v", currResult.File, currResult.Type, currResult.Error))
-			continue
-		}
 
-		if currResult.Changed && resourcesThatForceRedeployment.Has(currResult.File) {
-			forceRollout = true
-		}
-	}
+	// TODO - use labels/annotations to force a daemonset rollout
```

Comment:
IIRC this is a different TODO comment from what you had in your last PR in the same spot, and this is a TODO comment you want to keep, correct @adambkaplan ?

Reply:
Yes - we are passing `forceRollout` to a method that has been deprecated. We should use labels or annotations to drive rollouts.

Reply:
thanks for the confirmation